lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -263,7 +263,7 @@
 
     <div class="wrapper">
       <div class="container__block">
-        <h3 class="heading -beta inline-alt-text-color"><?php print t('Pics or It Didn’t Happen'); ?></h3>
+        <h3 class="heading -beta inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
```

Comment:
Not a biggie, but what's the reason behind this change? Kinda makes translation a little more difficult.